Objects/EventItem/EventManager.gd

```diff
@@ -17,10 +18,12 @@ func wait_to_spawn_item():
 	timer.start()
 
 func return_random_time() -> float:
-	var value = randi()%40+20
+	var value = randi()%2+1#%40+20
```

Comment:
I see You forgot to return previous value ;)

Reply:
Yes